hhs_hosp/delphi_hhs/constants.py

```diff
@@ -3,11 +3,12 @@
 
 CONFIRMED = "confirmed_admissions_covid_1d"
 SUM_CONF_SUSP = "sum_confirmed_suspected_admissions_covid_1d"
-
+POP_PROP="population_proportionate_variants"
```

Comment:
The basic structure and math looks good!

For signal names we want 
* a variant of each signal, and
* make it clear which signal is the base

so something like this:

```suggestion
CONFIRMED_PROP = "confirmed_admissions_covid_1d_prop"
SUM_CONF_SUSP_PROP = "sum_confirmed_suspected_admissions_covid_1d_prop"
```

& then carry that through to `make_signal`, etc

Reply:
Got it!